From 4d2ba80cd1b972478ec3821908f99e1f8a7b9809 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Thu, 26 Mar 2015 15:30:41 +0800
Subject: [PATCH] MLK-10472-2 ARM: imx_v7_defconfig: enable max7322 in default
 config

Enable max7322 extention gpio driver in config

(cherry-picked from commit: 575b7e4c2f85b08fb09d6a639adb404718b9b644)

Signed-off-by: Luwei Zhou <b45643@freescale.com>
Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/configs/imx_v7_defconfig | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index 8ff820b..653d140 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -2,6 +2,8 @@ CONFIG_KERNEL_LZO=y
 CONFIG_SYSVIPC=y
 CONFIG_NO_HZ=y
 CONFIG_HIGH_RES_TIMERS=y
+CONFIG_IKCONFIG=y
+CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_BUF_SHIFT=18
 CONFIG_CGROUPS=y
 CONFIG_RELAY=y
@@ -186,6 +188,7 @@ CONFIG_SPI=y
 CONFIG_SPI_GPIO=y
 CONFIG_SPI_IMX=y
 CONFIG_GPIO_SYSFS=y
+CONFIG_GPIO_MAX732X=y
 CONFIG_GPIO_74X164=y
 CONFIG_POWER_SUPPLY=y
 CONFIG_SABRESD_MAX8903=y
@@ -427,5 +430,3 @@ CONFIG_LIBCRC32C=m
 CONFIG_FONTS=y
 CONFIG_FONT_8x8=y
 CONFIG_FONT_8x16=y
-CONFIG_IKCONFIG=y
-CONFIG_IKCONFIG_PROC=y
-- 
1.8.0

